lib/tasks/null_fields_counter.rake

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+desc 'Count null fields in a column'
+task :count_null_fields, %i[model column] => :environment do |task, args|
+  abort 'Model or column param missing' if args[:column].blank? || args[:model].blank?
+  model = args[:model]
+  column = args[:column]
+  begin
+    count = model.constantize.where("#{column}": nil).count
```

Comment:
Let's change this to `model.constantize.where("#{column}": nil).size`  In general, we want to use `size` over `count` for performance considerations; it may not make much of a difference here, but it's a good habit to get into [(more detail here)](https://work.stevegrossi.com/2015/04/25/how-to-count-with-activerecord/)